app/src/metrics/README.md

```diff
@@ -0,0 +1,203 @@
+# Metrics Module
+
+This module provides tools for evaluating retrieval performance using recall and ranking metrics.
+
+## Overview
+
+The metrics module allows you to:
+1. Run evaluations with configurable parameters
+2. Compute recall@k as the retrieval effectiveness metric 
+3. Generate structured evaluation logs and CSV reports
+4. Track performance across different datasets
+
+## Directory Structure
+
+```
+metrics/
+├── __init__.py
+├── README.md
+├── cli.py
+├── evaluation/
+│   ├── __init__.py
+│   ├── batch.py
+│   ├── logging.py
+│   ├── metrics.py
+│   ├── results.py
+│   └── runner.py
+├── models/
+│   ├── __init__.py
+│   └── metrics.py
+├── utils/
+│   ├── __init__.py
+│   ├── timer.py
+│   └── jsonl_to_csv.py
```

Comment:
This is not particularly useful. We'd also have to keep it updated with code changes. I suggest removing.